learn/jbang/hello.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright 2024 The Enola <https://enola.dev> Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//JAVA 21
+//COMPILE_OPTIONS --enable-preview -source 21
```

Comment:
You can just do //PREVIEW and jbang will do these automatically for you

Reply:
Oh, cool! That indeed [seems to work](https://github.com/enola-dev/enola/pull/1001/commits/38e47f5c6224af8da3bba1f2019dd2bc3b90264c), but requires Mind 🧠 Reading to know about 🤣 as it does not appear to be documented - I've sent you https://github.com/jbangdev/jbang/pull/1894.